### ISSUE TITLE
Wrong type of options val in org.scalajs.dom.raw.HTMLSelectElement

### ISSUE THREAD

**Issue by jn73** [scala-js/scala-js-dom#120]
In org.scalajs.dom.raw.HTMLSelectElement the declaration of:

```scala
val options: js.Array[HTMLSelectElement] = js.native
```

Should be:

```scala
val options: js.Array[HTMLOptionElement] = js.native
```

**Comment by sjrd** [comment]
Thanks for the report.

**Comment by mseddon** [comment]
This one has already been fixed! (but unreleased) see: 40d6a71d4f8d2526a196744bb98f526d4f83ea1c
